### ISSUE TITLE
 Deployment fails on deNBI/Bielefeld

### ISSUE THREAD

**Issue by sneumann** [phnmnl/ng2-phenomenal-portal#22]
I just tried https://portaldev.phenomenal-h2020.eu/cloud-research-environment-setup in Bielefeld, and was unfortunately not very successful: I didn't get past the Verify.
OS credentials have been checked, local cloud-deploy-kubenow works nicely. Yours, Steffen

**Comment by sneumann** [comment]
Update: with RC file upload on portaldev I get past flavour selection, 
but fail on the actual deployment. 

```
TF_VAR_client_id=
TF_VAR_aws_access_key_id=
OS_PROJECT_ID=
OS_PROJECT_NAME=PhenoMeNal
TF_VAR_gce_project=
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0
100   388    0   388    0     0    169      0 --:--:--  0:00:02 --:--:--   169
  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0
100  415k  100  415k    0     0   159k      0  0:00:02  0:00:02 --:--:-- 1412k
Archive:  netifaces_wheelhouse.zip
   creating: netifaces_wheelhouse/
  inflating: netifaces_wheelhouse/netifaces-0.10.6-cp27-cp27m-manylinux1_i686.whl  
  inflating: netifaces_wheelhouse/netifaces-0.10.6-cp27-cp27m-manylinux1_x86_64.whl  
  inflating: netifaces_wheelhouse/netifaces-0.10.6-cp27-cp27mu-manylinux1_i686.whl  
  inflating: netifaces_wheelhouse/netifaces-0.10.6-cp27-cp27mu-manylinux1_x86_64.whl  
  inflating: netifaces_wheelhouse/netifaces-0.10.6-cp33-cp33m-manylinux1_i686.whl  
  inflating: netifaces_wheelhouse/netifaces-0.10.6-cp33-cp33m-manylinux1_x86_64.whl  
  inflating: netifaces_wheelhouse/netifaces-0.10.6-cp34-cp34m-manylinux1_i686.whl  
  inflating: netifaces_wheelhouse/netifaces-0.10.6-cp34-cp34m-manylinux1_x86_64.whl  
  inflating: netifaces_wheelhouse/netifaces-0.10.6-cp35-cp35m-manylinux1_i686.whl  
  inflating: netifaces_wheelhouse/netifaces-0.10.6-cp35-cp35m-manylinux1_x86_64.whl  
  inflating: netifaces_wheelhouse/netifaces-0.10.6-cp36-cp36m-manylinux1_i686.whl  
  inflating: netifaces_wheelhouse/netifaces-0.10.6-cp36-cp36m-manylinux1_x86_64.whl  
New python executable in /mnt/ecp/data/be_deployments_folder/TSI1522056798557/deploy/bin/python2
Also creating executable in /mnt/ecp/data/be_deployments_folder/TSI1522056798557/deploy/bin/python
Installing setuptools, pip, wheel...done.
Requirement already up-to-date: pip in ./deploy/lib/python2.7/site-packages
...
Collecting wrapt==1.10.11 (from -r /mnt/ecp/data/be_applications_folder/usr-73f4f5ad-ae70-458d-839c-f2e258677a17/cloud-deploy-kubenow.git/requirements_glance.txt (line 35))
Building wheels for collected packages: functools32, prettytable
  Running setup.py bdist_wheel for functools32: started
  Running setup.py bdist_wheel for functools32: finished with status 'done'
  Stored in directory: /home/centos/.cache/pip/wheels/b5/83/8b/2de165caa2da68be141fbf343fd291fbcae73094c213ba3f5c
  Running setup.py bdist_wheel for prettytable: started
  Running setup.py bdist_wheel for prettytable: finished with status 'done'
  Stored in directory: /home/centos/.cache/pip/wheels/0d/08/4d/38d1751a9e86f7f568d619e8e0662fb85241d17df8c90c5d8a
Successfully built functools32 prettytable
Installing collected packages: asn1crypto, Babel, certifi, cffi, chardet, cryptography, debtcollector, enum34, funcsigs, functools32, idna, ipaddress, iso8601, jsonpatch, jsonpointer, jsonschema, keystoneauth1, monotonic, netaddr, oslo.i18n, oslo.utils, pbr, prettytable, pycparser, pyOpenSSL, pyparsing, python-glanceclient, pytz, requests, six, stevedore, urllib3, warlock, wrapt
Successfully installed Babel-2.5.3 asn1crypto-0.24.0 certifi-2018.1.18 cffi-1.11.4 chardet-3.0.4 cryptography-2.1.4 debtcollector-1.19.0 enum34-1.1.6 funcsigs-1.0.2 functools32-3.2.3.post2 idna-2.6 ipaddress-1.0.19 iso8601-0.1.12 jsonpatch-1.21 jsonpointer-2.0 jsonschema-2.6.0 keystoneauth1-3.4.0 monotonic-1.4 netaddr-0.7.19 oslo.i18n-3.19.0 oslo.utils-3.35.0 pbr-3.1.1 prettytable-0.7.2 pyOpenSSL-17.5.0 pycparser-2.18 pyparsing-2.2.0 python-glanceclient-2.9.1 pytz-2017.3 requests-2.18.4 six-1.11.0 stevedore-1.28.0 urllib3-1.22 warlock-1.3.0 wrapt-1.10.11
Started script image-create-openstack
List images available in OpenStack...
Expecting to find domain in project - the server could not comply with the request since it is either malformed or otherwise incorrect. The client is assumed to be in error. (HTTP 400) (Request-ID: req-5f50cacd-144e-4036-9b48-887fc8447019)
TF_VAR_client_id=
TF_VAR_aws_access_key_id=
OS_PROJECT_ID=
OS_PROJECT_NAME=PhenoMeNal
TF_VAR_gce_project=
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 14910    0  2070  100 12840   4281  26560 --:--:-- --:--:-- --:--:-- 26583
{"ok":true,"file":{"id":"F9W3SFAKX","created":1522056817,"timestamp":1522056817,"name":"output-TSI1522056798557.log","title":"output-TSI1522056798557","mimetype":"text\/plain","filetype":"shell","pretty_type":"Shell","user":"U6NKZ1K09","editable":true,"size":12160,"mode":"snippet","is_external":false,"external_type":"","is_public":false,"public_url_shared":false,"display_as_bot":false,"username":"","url_private":"https:\/\/files.slack.com\/files-pri\/T0E4T7X45-F9W3SFAKX\/output-tsi1522056798557.log","url_private_download":"https:\/\/files.slack.com\/files-pri\/T0E4T7X45-F9W3SFAKX\/download\/output-tsi1522056798557.log","permalink":"https:\/\/phenomenal-h2020.slack.com\/files\/U6NKZ1K09\/F9W3SFAKX\/output-tsi1522056798557.log","permalink_public":"https:\/\/slack-files.com\/T0E4T7X45-F9W3SFAKX-9d7d5ab866","edit_link":"https:\/\/phenomenal-h2020.slack.com\/files\/U6NKZ1K09\/F9W3SFAKX\/output-tsi1522056798557.log\/edit","preview":"TF_VAR_client_id=\nTF_VAR_aws_access_key_id=\nOS_PROJECT_ID=\nOS_PROJECT_NAME=PhenoMeNal\nTF_VAR_gce_project=","preview_highlight":"<div class=\"CodeMirror cm-s-default CodeMirrorServer\" oncopy=\"if(event.clipboardData){event.clipboardData.setData('text\/plain',window.getSelection().toString().replace(\/\\u200b\/g,''));event.preventDefault();event.stopPropagation();}\">\n<div class=\"CodeMirror-code\">\n<div><pre><span class=\"cm-def\">TF_VAR_client_id<\/span><span class=\"cm-operator\">=<\/span><\/pre><\/div>\n<div><pre><span class=\"cm-def\">TF_VAR_aws_access_key_id<\/span><span class=\"cm-operator\">=<\/span><\/pre><\/div>\n<div><pre><span class=\"cm-def\">OS_PROJECT_ID<\/span><span class=\"cm-operator\">=<\/span><\/pre><\/div>\n<div><pre><span class=\"cm-def\">OS_PROJECT_NAME<\/span><span class=\"cm-operator\">=<\/span>PhenoMeNal<\/pre><\/div>\n<div><pre><span class=\"cm-def\">TF_VAR_gce_project<\/span><span class=\"cm-operator\">=<\/span><\/pre><\/div>\n<\/div>\n<\/div>\n","lines":116,"lines_more":111,"preview_is_truncated":true,"channels":[],"groups":["G640MEZAQ","G640MEZAQ"],"ims":[],"comments_count":0}}
```

**Comment by andersla** [comment]
The issue was a VM flavor that enabled swap. Swap is not allowed on Kubernetes nodes and therefore preflight-check failed.
A fix setting swapoff on all nodes before running Kubeadm resolved issue phnmnl/cloud-deploy-Kube#d985fc2bd7793d42b67273665916e23c5c99c835
